docs/messaging/messaging-spans.md

```diff
@@ -389,6 +389,20 @@ If a messaging operation involved multiple network calls (for example retries),
 
 
 
+The following attributes can be important for making sampling decisions
```

Comment:
Any reason for leaving out `messaging.destination.anonymous` and `messaging.client.id`, which could be known at the time a span starts? Could those possibly be sampling relevant?

Reply:
Discussed in the workgroup meeting: `messaging.destination.anonymous` will not be declared stable and can be added to sampling relevant attributes later if needed, `messaging.client.id` is not useful for sampling.